packages/connect-ui/src/index.tsx

```diff
@@ -86,12 +87,16 @@ export const ConnectUI = ({ postMessage, clearLegacyView }: ConnectUIProps) => {
         // notifications
         const notifications: React.ReactNode[] = [];
         messages.forEach(message => {
-            if (message?.type === 'popup-handshake' && message?.payload?.transport?.outdated) {
-                notifications.push(<BridgeUpdateNotification key="bridge-outdated" />);
+            if (message?.type === 'popup-handshake') {
+                if (message.payload?.transport?.outdated) {
+                    notifications.push(<BridgeUpdateNotification key="bridge-outdated" />);
+                }
             } else if (message?.type === UI_REQUEST.FIRMWARE_OUTDATED) {
                 notifications.push(<FirmwareUpdateNotification key={message.type} />);
             } else if (message?.type === UI_REQUEST.DEVICE_NEEDS_BACKUP) {
                 notifications.push(<BackupNotification key={message.type} />);
+            } else if (message?.type === 'phishing-domain') {
+                notifications.push(<SuspiciousOriginNotification key="suspicious origin" />);
```

Comment:
nit: to keep it same as previous notifications
```suggestion
                notifications.push(<SuspiciousOriginNotification key="suspicious-origin" />);
```

Reply:
640a9cf3b6f34e8a421365b2d97912055acf0589